main.py

```diff
@@ -2,6 +2,36 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres_data = [
+        ("Western",),
+        ("Action",),
+        ("Dramma",)
```

Comment:
There is a typo in the genre name 'Dramma'. It should be 'Drama'. This is corrected later in the script, but it's better to fix it in the initial data to avoid unnecessary updates.